[DCP][BE] Apply ufmt to DCP and turn on lintrunner for DCP

fegin opened the pull request:
Stack from [ghstack](https://github.com/ezyang/ghstack) (oldest at bottom):
* #113209
* #110662
* #115650
* #115649
* #115648
* __->__ #115302
* #115523

No logic change. Just typing and ufmt.

Differential Revision: [D51914982](https://our.internmc.facebook.com/intern/diff/D51914982/)

cc @mrshenli @pritamdamania87 @zhaojuanmao @satgera @rohan-varma @gqchen @aazzolini @osalpekar @jiayisuse @H-Huang @kwen2501 @awgu @penguinwu @XilunWu @wanchaol @fduwjj @wz337 @tianyu-l @wconstab @yf225 @kiukchung @lucasllc

fegin committed:
[DCP][BE] Apply ufmt to DCP and turn on lintrunner for DCP

No logic change. Just typing and ufmt.

Differential Revision: [D51914982](https://our.internmc.facebook.com/intern/diff/D51914982/)

[ghstack-poisoned]

fegin committed:
Update on "[DCP][BE] Apply ufmt to DCP and turn on lintrunner for DCP"


No logic change. Just typing and ufmt.

Differential Revision: [D51914982](https://our.internmc.facebook.com/intern/diff/D51914982/)

cc H-Huang awgu kwen2501 wanchaol fduwjj wz337 wconstab mrshenli zhaojuanmao rohan-varma kiukchung lucasllc XilunWu tianyu-l yf225

[ghstack-poisoned]

fegin committed:
Update on "[DCP][BE] Apply ufmt to DCP and turn on lintrunner for DCP"


No logic change. Just typing and ufmt.

Differential Revision: [D51914982](https://our.internmc.facebook.com/intern/diff/D51914982/)

cc H-Huang awgu kwen2501 wanchaol fduwjj wz337 wconstab mrshenli zhaojuanmao rohan-varma kiukchung lucasllc XilunWu tianyu-l yf225

[ghstack-poisoned]

XilunWu commented:
plan to merge it or it's still WIP?

fegin committed:
Update on "[DCP][BE] Apply ufmt to DCP and turn on lintrunner for DCP"


No logic change. Just typing and ufmt.

Differential Revision: [D51914982](https://our.internmc.facebook.com/intern/diff/D51914982/)

cc H-Huang awgu kwen2501 wanchaol fduwjj wz337 wconstab mrshenli zhaojuanmao rohan-varma kiukchung lucasllc XilunWu tianyu-l yf225

[ghstack-poisoned]

fegin committed:
Update on "[DCP][BE] Apply ufmt to DCP and turn on lintrunner for DCP"


No logic change. Just typing and ufmt.

Differential Revision: [D51914982](https://our.internmc.facebook.com/intern/diff/D51914982/)

cc H-Huang awgu kwen2501 wanchaol fduwjj wz337 wconstab mrshenli zhaojuanmao rohan-varma kiukchung lucasllc XilunWu tianyu-l yf225

[ghstack-poisoned]

fegin committed:
Update on "[DCP][BE] Apply ufmt to DCP and turn on lintrunner for DCP"


No logic change. Just typing and ufmt.

Differential Revision: [D51914982](https://our.internmc.facebook.com/intern/diff/D51914982/)

cc mrshenli pritamdamania87 zhaojuanmao satgera rohan-varma gqchen aazzolini osalpekar jiayisuse H-Huang kwen2501 awgu penguinwu XilunWu wanchaol fduwjj wz337 tianyu-l wconstab yf225 kiukchung lucasllc

[ghstack-poisoned]

fegin committed:
Update on "[DCP][BE] Apply ufmt to DCP and turn on lintrunner for DCP"


No logic change. Just typing and ufmt.

Differential Revision: [D51914982](https://our.internmc.facebook.com/intern/diff/D51914982/)

cc mrshenli pritamdamania87 zhaojuanmao satgera rohan-varma gqchen aazzolini osalpekar jiayisuse H-Huang kwen2501 awgu penguinwu XilunWu wanchaol fduwjj wz337 tianyu-l wconstab yf225 kiukchung lucasllc

[ghstack-poisoned]

fegin committed:
Update on "[DCP][BE] Apply ufmt to DCP and turn on lintrunner for DCP"


No logic change. Just typing and ufmt.

Differential Revision: [D51914982](https://our.internmc.facebook.com/intern/diff/D51914982/)

cc mrshenli pritamdamania87 zhaojuanmao satgera rohan-varma gqchen aazzolini osalpekar jiayisuse H-Huang kwen2501 awgu penguinwu XilunWu wanchaol fduwjj wz337 tianyu-l wconstab yf225 kiukchung lucasllc

[ghstack-poisoned]

fegin committed:
Update on "[DCP][BE] Apply ufmt to DCP and turn on lintrunner for DCP"


No logic change. Just typing and ufmt.

Differential Revision: [D51914982](https://our.internmc.facebook.com/intern/diff/D51914982/)

cc mrshenli pritamdamania87 zhaojuanmao satgera rohan-varma gqchen aazzolini osalpekar jiayisuse H-Huang kwen2501 awgu penguinwu XilunWu wanchaol fduwjj wz337 tianyu-l wconstab yf225 kiukchung lucasllc

[ghstack-poisoned]

fegin committed:
Update on "[DCP][BE] Apply ufmt to DCP and turn on lintrunner for DCP"


No logic change. Just typing and ufmt.

Differential Revision: [D51914982](https://our.internmc.facebook.com/intern/diff/D51914982/)

cc mrshenli pritamdamania87 zhaojuanmao satgera rohan-varma gqchen aazzolini osalpekar jiayisuse H-Huang kwen2501 awgu penguinwu XilunWu wanchaol fduwjj wz337 tianyu-l wconstab yf225 kiukchung lucasllc

[ghstack-poisoned]

fegin committed:
Update on "[DCP][BE] Apply ufmt to DCP and turn on lintrunner for DCP"


No logic change. Just typing and ufmt.

Differential Revision: [D51914982](https://our.internmc.facebook.com/intern/diff/D51914982/)

cc mrshenli pritamdamania87 zhaojuanmao satgera rohan-varma gqchen aazzolini osalpekar jiayisuse H-Huang kwen2501 awgu penguinwu XilunWu wanchaol fduwjj wz337 tianyu-l wconstab yf225 kiukchung lucasllc

[ghstack-poisoned]

fegin committed:
Update on "[DCP][BE] Apply ufmt to DCP and turn on lintrunner for DCP"


No logic change. Just typing and ufmt.

Differential Revision: [D51914982](https://our.internmc.facebook.com/intern/diff/D51914982/)

cc mrshenli pritamdamania87 zhaojuanmao satgera rohan-varma gqchen aazzolini osalpekar jiayisuse H-Huang kwen2501 awgu penguinwu XilunWu wanchaol fduwjj wz337 tianyu-l wconstab yf225 kiukchung lucasllc

[ghstack-poisoned]

wz337 reviewed (approved):
LGTM! Thanks for cleaning this up!

LucasLLC reviewed (approved):
Awesome!! Thanks @fegin !!

fegin committed:
Update on "[DCP][BE] Apply ufmt to DCP and turn on lintrunner for DCP"


No logic change. Just typing and ufmt.

Differential Revision: [D51914982](https://our.internmc.facebook.com/intern/diff/D51914982/)

cc mrshenli pritamdamania87 zhaojuanmao satgera rohan-varma gqchen aazzolini osalpekar jiayisuse H-Huang kwen2501 awgu penguinwu XilunWu wanchaol fduwjj wz337 tianyu-l wconstab yf225 kiukchung lucasllc

[ghstack-poisoned]
